python/GangaDirac/Dirac.ini

```diff
@@ -4,6 +4,12 @@ RUNTIME_PATH = GangaDirac
 
 ReleaseNotes = False
 
+[defaults_GridCommand]
+info = /cvmfs/ganga.cern.ch/dirac_ui/scripts/dirac-proxy-info
+init = /cvmfs/ganga.cern.ch/dirac_ui/scripts/dirac-proxy-init -g gridpp_user -M
+[DIRAC]
+DiracEnvFile = GangaDirac/cvmfs_DIRAC_env
```

Comment:
I'm confused - is the envfile in this PR different from:

/cvmfs/ganga.cern.ch/dirac_ui/envfile

? If so, I'll just update the CVMFS one and DiracEnvFile should point to that.